packages/docs/celo-gold-holder-guide/voting-validators.md

```diff
@@ -0,0 +1,81 @@
+# Voting on Validator Groups
+
+## Proof of Stake Election Essentials
+
+Validators play a critical role in the Celo protocol, determining which transactions get applied and producing new blocks. Selecting organizations that operate well-run infrastructure to perform this role effectively is essential for Celo's long-term success.
+
+The Celo community makes these decisions through elections for [Validator Groups](../celo-codebase/protocol/proof-of-stake/validator-groups.md), intermediaries between voters and validators. Every validator group has an ordered list of up to 5 candidate validators. Some organizations may operate a group with their own validators in it; some may operate a group to which they have added validators run by others.
+
+[Validator elections](../celo-codebase/protocol/proof-of-stake/validator-elections.md) are held every epoch (approximately once per day). All 100 slots are available at each election. Validators are selected [in proportion](../celo-codebase/protocol/proof-of-stake/validator-elections.md#Running-the-Election) to votes received for each Validator Group.
```

Comment:
I am not sure what "All 100 slots are available at each election." is supposed to communicate in this context. Is the idea to distinguish that this is not like e.g. the US Senate where only some seats are available each time?

Reply:
exactly. Thoughts on better wording than `up for election`?